config/locales/en/models/claim.yml

```diff
@@ -262,6 +262,21 @@ en:
               too_long_before_earliest_reporder: The graduated fee date cannot be no earlier than two years before the earliest representation order date
             quantity:
               numericality: Enter a valid quantity for the graduated fee
+        fee/hardship_fee:
+          attributes:
+            blank: Add a hardship fee
+            base:
+              uneditable_state: You cannot edit a claim that is not in draft state
+            fee_type:
+              blank: Choose a type for the hardship fee
+            quantity:
```

Comment:
Should `quantity` have a key:value pair for the attribute `:item_max_amount`
The validator `def validate_presence_and_numericality_govuk_formbuilder` validates against `:blank`, `:numericality` and `:item_max_amount`